packs/oam-app-controller-0.0.3/pack.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "oam-app-controller",
+    "displayName": "OAM App Controller",
+    "annotations": {
+        "description": "OAM Application Controller",
+        "contributor": "spectrocloud"
+    },
+    "version": "0.0.3",
```

Comment:
Going forward, we should probably just use patch versions any time any chart is updated.